packages/components/src/Card/Card.mdx

```diff
@@ -112,3 +114,80 @@ Card titles should be sentence-cased.
     </Content>
   </Card>
 </Playground>
+
+### Header with action
+
+Use the header action when there's an action tied directly to the Card's
+contents, such as adding more items. It comes with pre-defined styling and
+layout to standardize how Card actions look across the product.
+
+<Playground>
+  <Card
+    header={{
+      title: "Banana varieties",
+      action: {
+        label: "New Banana",
+        type: "primary",
```

Comment:
I'd remove this. make sure the copy-paster doesn't think that you need to have this.

Reply:
Good call, we should _default_ to showing the default 😄